### ISSUE TITLE
Can't open SelectBoxIt in 

### ISSUE THREAD

**Issue by mhairston** [gfranko/jquery.selectBoxIt.js#387]
I am trying to get SelectBoxIt to work in VoiceOver/Safari (High Sierra 10.13.5, Safari 11.1.1, VoiceOver v8 when checked in VoiceOver Utility)

## Tested most recent SelectBoxIt on project home (v3.8.2 acc. to package.json):

* No screen reader, Chrome or Safari:
  * Plain select can be opened by spacebar, up or down arrow.
  * SelectBoxIt: can be opened by up or down arrow, but not spacebar.
* VoiceOver in Safari:
  * Plain select: can be opened by ctrl-opt-space, but not arrows.
  * SelectBoxIt: seemingly cannot be opened.

I cannot seem to open SelectBoxIt while using VoiceOver -- I noted that in a previous issue from 2013, you mentioned testing in VoiceOver. Is there a key trigger that I am missing?

If I hit space on a SelectBoxIt when the original select is visible (which in practice it won't be), it triggers the _original_ menu to open.
